modules/fedimint-ln-client/src/pay.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(deprecated)]
```

Comment:
A crate level `allow` can let multiple new ones slip it, and also no one will remember to clean it up after it is no longer needed. IMO, we should put it on every line that is using the deprecated symbol.

Reply:
So I tried that initially, and I could only get the usage warning to go away, but for some reason when adding `#[allow(deprecated)]` on the declaration, the derive statement on the enum causes problems and the `#[allow(deprecated)]` macro is ignored. 

Maybe that's good enough?

Reply:
Let's try what we for start.